### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by tamzi** [droidconKE/droidconKE2020App#4]
Do we need better documentation with everything for the app?

**Comment by JabezNzomo99** [comment]
Yes, i think it would be great to have a detailed documentation of the architecture, technologies and flow of the app

**Comment by jamesnyakush** [comment]
It would be great so that in future after the events someone can refer to it and catch up what guys were really doing

**Comment by jamesnyakush** [comment]
I have realized that @wangerekaharun has a tendency of revisiting the code when he tries to explain i think we will be making his life easy so that people can get what he always says quickly and have a place to refer

**Comment by wangerekaharun** [comment]
Yes in full support of this

**Comment by chepsi** [comment]
Also, the need for proper documentation is so that community members who want to learn, can read good code which can in turn help them write better code, understand how features were implemented and also act as a guide on how to properly document projects.

**Comment by jamesnyakush** [comment]
Great  this is the only way forward just a suggestion we should write every bit like how modularization , Dependency injection was done like topic-based how safe API  was handled and the likes

**Comment by tamzi** [comment]
thsi seems like it will be a continous process

**Comment by wangerekaharun** [comment]
Yes this issue remains open

**Comment by odaridavid** [comment]
On the Readme,there seems to be duplication on references and could use slight grammatical and content formatting improvements.

**Comment by wangerekaharun** [comment]
> On the Readme,there seems to be duplication on references and could use slight grammatical and content formatting improvements.

Yes its Work In Progress

**Comment by KelynPNjeri** [comment]
I would really love to help, can you please point me on the best place to start.

**Comment by wangerekaharun** [comment]
@KelynPNjeri How would you like to help with this, sorry it's been long.

**Comment by KelynPNjeri** [comment]
@wangerekaharun  I am kind of a beginner. So what tasks would you recommend?

**Comment by wangerekaharun** [comment]
> @wangerekaharun I am kind of a beginner. So what tasks would you recommend?

I would recommend you first take time to go through the project and familiarize yourself first then you can be in a better position to pick up a task once its raised. Or even you could find something that we are missing out and help in fixing it.
